internal/mkcw/archive.go

```diff
@@ -70,8 +71,6 @@ func (c chainRetrievalError) Error() string {
 
 // Archive generates a WorkloadConfig for a specified directory and produces a
 // tar archive of a container image's rootfs with the expected contents.
-// The input directory will have a ".krun_config.json" file added to it while
-// this function is running, but it will be removed on completion.
```

Comment:
This comment was no longer correct as of #5224, but I forgot to drop it during that one.